### ISSUE TITLE
Circle CI Lint Broken

### ISSUE THREAD

**Issue by sleepypioneer** [opensourcediversity/opensourcediversity.org#142]
Currently, the linter appears to not be able to check out the code, which means this check fails. At a glance it seems to be regarding the publickey used, I am also a bit surprised to see it print this out in the log but perhaps that is not an issue.

```
Using SSH Config Dir '/home/circleci/.ssh'
git version 2.37.1
Cloning git repository
Cloning into '.'...
Warning: Permanently added the ECDSA host key for IP address '140.82.113.3' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository
```

It appears to have been the case since circa 3 months, and perhaps since this was added:

![image](https://user-images.githubusercontent.com/23617373/198887728-7c898822-9c5f-47fe-ae2a-c1c812dbcce4.png)

**Comment by Roshanjossey** [comment]
I think circle CI used to work at some point because of logs in issues like https://github.com/opensourcediversity/opensourcediversity.org/issues/123

How about dropping circle CI and using GitHub actions for linting instead? We could use the same tool (tidy)

**Comment by Roshanjossey** [comment]
Fixed in #154
